test/controllers/IndexControllerSpec.scala

```diff
@@ -34,11 +33,8 @@ class IndexControllerSpec extends SpecBase {
 
         val result = route(application, request).value
 
-        val view = application.injector.instanceOf[IndexView]
+        status(result) mustEqual SEE_OTHER
```

Comment:
is it worth checking where it is redirecting?

```
headers(result) must contain ("Location" -> routes.YourAccountController.onPageLoad.url)
```

I am not usually a fan of how  must contain  fails (incoherent for things like a list of case classes where there can be issues of time causing problems), but in this case it is not too bad